lib/components/Widgets/Info/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Icon } from "@/components/Utilities/Icon"
+import BriefcaseIcon from "@/icons/Briefcase"
+import OfficeIcon from "@/icons/Office"
+import { forwardRef } from "react"
+
+const iconsMap = {
```

Comment:
Do we want to control what can be passed? Why can't we allow passing any icon available?